lib/auth/httpfallback.go

```diff
@@ -125,6 +96,7 @@ func (c *Client) DeleteRole(ctx context.Context, name string) error {
 	return trace.Wrap(err)
 }
 
+// DELETE IN 8.0
```

Comment:
Can this be deleted?

Reply:
Yup, reverted a commit that accidentally deleted this comment but I think we're good to delete it.